### PR TITLE
this is a tdd challenges homework

### PR DESCRIPTION
**Pull request by Yjohn**
please check my homework asap

### CODE REVIEW

File: 01-exercises/roman-converter/convert-to-old-roman.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+function convertToOldRoman(number) {
+    // ToDo: do the full implementation using TDD
+    return 'I';
```
Comment (ash-leigh) [inline 01-exercises/roman-converter/convert-to-old-roman.test.js]: I like the ToDo comment :) 
So the way you could implement this is to start with two arrays - one for your Arabic numbers and one for your old Roman numerals:
```
var arabicNumbers = [1, 5, 10, 50, 100, 500, 100];
var oldRomanNumerals = ['I', 'V', 'X', 'L', 'C', 'D', 'M'];
```
Starting with these your notice that the thing the 1 has in common with the 'I' is that they are both at the 0 index of their own array so:
```
arabicNumbers[0] = 1
```
and
```
oldRomanNumerals[0] = 'I'
```
So in your function you can use the Arabic number you pass in as an argument to find the index at which the old Roman Numeral is at in its array. - Make sense?

Remember to write your edge case tests too - so for example what would happen if I passed 2 in as an argument?

File: 01-exercises/roman-converter/convert-to-new-roman.test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+function convertToNewRoman(number) {
+    // ToDo: do the proper implementation using TDD
+    return 'I';
+}
+
+/* write the tests */
+test('convertToNewRoman should return I for 1', function() {
+    var result = convertToNewRoman(1);
+    expect(result).toBe('I');
+});
```
Comment (ash-leigh) [inline 01-exercises/roman-converter/convert-to-new-roman.test.js]: You can get a solution to this in a very similar way to the old roman numerals one but with changing the values in the array. 

I'm going to google to see if this is the only way it can be done so you do it too and we can compare on Saturday.

File: add-numbers.test.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+test('add numbers', function(){
+    // var result = add(2, 1, 3);
+    // expect(result).toEqual(6);
+    expect(add(4,2,4)).toEqual(10);
+});
+
+
+test("string", function(){})
+
+
```
Comment (ash-leigh) [inline add-numbers.test.js]: This is a great solution Yohannes - try and add some edge cases like what would happen if I added a string?

File: find-needle-advanced.test.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+function findMultipleNeedles(haystackArray) {
+    // ToDo: Implement this function to make all the tests below pass
+    // DO NOT Change the tests (you can add more tests though)
+
+    var result=[];
```
Comment (ash-leigh) [inline find-needle-advanced.test.js]: formatting is a little off here - careful with indentation
